exercises/practice/roman-numerals/.meta/config.json

```diff
@@ -0,0 +1,20 @@
+{
+    "authors": [
+        "axtens"
+    ],
+    "files": {
+        "solution": [
+            "src/roman-numerals.cob"
+        ],
+        "test": [
+            "test.sh",
```

Comment:
The test.sh shouldn't be in the "test" key. They should be in the "invalidator" key.